04turnOver.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+* 2)	Create a fuction which turns int number, e.g. 1234 -> 4321
+*/
+
+#include <iostream>
+
+using namespace std;
+
+void printDescription();
+int inputNumber(int number, string s);
+int turnNumber(int numberInit);
+void printNumberTurned(int numberTurned);
+
+int main()
+{
+	int numberInit = 0, numberTurned;
+
+	printDescription();
+
+	numberInit = inputNumber(numberInit, "Enter your number: ");

```

Comment:
You use `inputNumber` only once so `'Enter your number: '` can use into function

Reply:
Thank you! I will remember it. Shall I correct the code or leave it without changes?